i18n-hygiene.gemspec

```
@@ -8,11 +8,10 @@ Gem::Specification.new do |s|
   s.email = "dev@theconversation.edu.au"
   s.files = `git ls-files -- lib/*`.split("\n")
   s.homepage = 'https://github.com/conversation/i18n-hygiene'
-  s.has_rdoc = true
   s.extra_rdoc_files = ["README.md"]
   s.rdoc_options << "--title" << "i18n-hygiene documentation" << "--main" << "README.md"
 
-  s.add_dependency 'i18n', ['~> 0.6', '>= 0.6.9']
+  s.add_dependency 'i18n', '>= 1.6', '< 2'
```

Increasing `i18n` constraint to  1.x versions may affect application dependencies relying on previous major versions of `i18n`.

Since `i18n-hygiene` already requires at least `i18n` 0.6.9, what do you think on relaxing `i18n` requirement to accept anything above 0.6.9?

```suggestion
  s.add_dependency 'i18n', '>= 0.6.9'
```

---

I think I'd be fine with that - I was trying to avoid allowing a new major version that could have breaking changes that affect this gem, but still allowed v1.6.

---

We could do `'>= 0.6.9', '< 2'` for that, I suppose

---

`'>= 0.6.9', '< 2'` would be 👍

---

Done 👍